orthoinference/src/main/java/org/reactome/orthoinference/InferEWAS.java

```diff
@@ -0,0 +1,354 @@
+package org.reactome.orthoinference;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaClass;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.gk.schema.SchemaClass;
+
+public class InferEWAS {
+
+	private static MySQLAdaptor dba;
+	static boolean refDb = false;
+	private static String altRefDBId;
+	private static GKInstance instanceEdit;
+	private static GKInstance ensgDbInst;
+	private static GKInstance enspDbInst;
+	private static GKInstance alternateDbInst;
+	private static GKInstance uniprotDbInst;
+	private static GKInstance speciesInst;
+	private static HashMap<String, String[]> homologueMappings = new HashMap<String,String[]>();
+	private static HashMap<String, ArrayList<String>> ensgMappings = new HashMap<String,ArrayList<String>>();
+	private static HashMap<String, GKInstance> seenRGP = new HashMap<String,GKInstance>();
+	private static HashMap<String,GKInstance> ewasIdenticals = new HashMap<String,GKInstance>();
+	private static HashMap<String,GKInstance> residueIdenticals = new HashMap<String,GKInstance>();
+
+    // Creates an array of inferred EWAS instances from the homologue mappings file (hsap_species_mapping.txt).
+	@SuppressWarnings("unchecked")
+	public static ArrayList<GKInstance> ewasInferrer(GKInstance ewasInst) throws InvalidAttributeException, Exception
```

Comment:
Naming convention: methods are usually named as _verbs_. So this would be "inferEwas".